modules/installation-aws-permissions.adoc

```diff
@@ -212,19 +207,24 @@ If you have not created a load balancer in your AWS account, the IAM user also r
 * `s3:PutObject`
 * `s3:PutObjectAcl`
 * `s3:PutObjectTagging`
+
+[NOTE]
+=====
+If you set the `preserveBootstrapIgnition` AWS parameter in your `install-config.yaml` file, the `s3:DeleteObject` permission is optional.
+=====
 ====
 
 .Required permissions to delete base cluster resources
 [%collapsible]
 ====
 * `autoscaling:DescribeAutoScalingGroups`
-* `ec2:DeletePlacementGroup`
 * `ec2:DeleteNetworkInterface`
+* `ec2:DeletePlacementGroup`
 * `ec2:DeleteVolume`
 * `elasticloadbalancing:DeleteTargetGroup`
 * `elasticloadbalancing:DescribeTargetGroups`
-* `iam:DeleteAccessKey`
```

Comment:
`iam:DeleteAccessKey` should NOT be removed from `.Required permissions to delete base cluster resources`

Reply:
Readded `iam:DeleteAccessKey` permission to said section.